template/src/functions.php

```diff
@@ -12,6 +12,9 @@
 define('COMMIT', '<commit>');
 define('BRANCH', '<branch_ref>');
 
+define('WEBSITE_TYPE', '<%= type %>');
```

Comment:
Wordpress_type misschien beter?

Reply:
Al is wordpress type: wordpress ook een beetje gek 🤔 misschien default vs api